components/todolist.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react";
+import { FlatList, StyleSheet, View } from "react-native";
+import Header from "./header";
+import TaskList from "./taskList";
+import TaskInput from "./taskInput";
+
+class ToDoList extends Component {
+  state = {
+    currentId: 3,
```

Comment:
Remove this id
you can get item index in renderItem method

`  renderItem={({ item }) => ( <Task  id={item.index}`